docs/src/kubectl-plugin.md

```diff
@@ -780,6 +781,39 @@ kubectl cnpg logs cluster cluster-example --output my-cluster.log
 Successfully written logs to "my-cluster.log"
 ```
 
+#### Pretty
+
+The `pretty` sub-command reads a log stream from standard input and formats it
+into a human-readable output, and attempts to sort the entries by timestamp.
+
+It is intended to be used in combination with `kubectl cnpg logs cluster`, as
+shown in the following example:
```

Comment:
last week chatting with Gabriele, I had the impression we wanted to get rid of `Kubectl CNPG logs cluster` in favor of `stern`.
Not the plan anymore?
Or, to be done in a future PR?

Reply:
Yes, #5780 is about that.
I agree with recommending `stern`, but we should fix what we have if possible, too.